frontend/shared/components/src/members/components/view/ViewMemberAccountsBox.vue

```diff
@@ -54,25 +54,32 @@
             </STList>
         </template>
 
-        <template v-if="member.patchedMember.details.securityCode">
+        <div v-if="securityCode" class="hover-box container">
             <hr>
-            <h2>Beveiligingscode</h2>
+            <h2 class="style-with-button">
+                <div>Beveiligingscode</div>
+                <div>
```

Comment:
De knop mag enkel zichtbaar zijn voor de personen met de juiste toegangsrechten om deze opnieuw in te stellen. Hiervoor zou ik voorlopig enkel personen met full permissions tot de member toelaten om deze te laten wissen